test/integration_test.js

```diff
@@ -35,77 +99,3 @@ function clearConnectorFolder() {
         console.warn(e);
     }
 }
-
-// Start e2e
-chromedriver.start(); // on port 9515
-process.on('exit', chromedriver.stop);
-
-describe('plotly database connector', function Spec() {
```

Comment:
@n-riesco Seems to have been removed in this PR..

Reply:
Answered here: https://github.com/plotly/plotly-database-connector/pull/180#pullrequestreview-64663339